ERCS/erc-chain-identifiers.md

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
```suggestion
---
eip: 7785
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.